engine/apps/api/views/channel_filter.py

```diff
@@ -33,6 +34,14 @@ class ChannelFilterFilter(ModelFieldFilterMixin, filters.FilterSet):
     )
 
 
+@extend_schema_view(
+    list=extend_schema(responses=ChannelFilterSerializer),
+    create=extend_schema(request=ChannelFilterCreateSerializer, responses=ChannelFilterUpdateResponseSerializer),
+    update=extend_schema(request=ChannelFilterUpdateSerializer, responses=ChannelFilterUpdateResponseSerializer),
+    partial_update=extend_schema(
+        request=ChannelFilterUpdateSerializer, responses=ChannelFilterUpdateResponseSerializer
+    ),
+)
```

Comment:
this was previously here, it's only needed for our openapi schema. Needed to add it back with the re-addition of [this](https://github.com/grafana/oncall/pull/5228/files#diff-f4a0cc363f3adeab2d0546fd6290020d7f13cb21956c8293ff2530f8d07a974bR146) (serialized/deserialized types of `slack_channel` are different (`str` vs nested `dict`), hence the need for this)